query-advisor/insights/index.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: 'Query Advisor Insights'
+backlink_href: /docs/query-advisor
+backlink_title: 'Query Advisor'
+---
+
+Query Advisor automatically detects common Postgres query optimization opportunities by analyzing EXPLAIN plans for known performance anti-patterns. This section documents each supported insight type, including detection criteria, optimization strategies, and implementation approaches.
+
+## How insights work
+
+Query Advisor uses a **pattern library approach** to identify optimization opportunities:
+
+1. **Pattern detection**: Analyzes EXPLAIN plans for specific performance anti-patterns
+2. **Validation**: Verifies that alternative query plans are likely available
+3. **Solution generation**: Provides tested query modifications or configuration changes
+4. **Integration**: Seamlessly applies suggestions through pganalyze Workbooks
+
+All insights are based on **deterministic algorithms** derived from Postgres planner behavior - no machine learning involved.
+
+## Currently supported insights
+
+### Inefficient Nested Loops
+
+Detects when Postgres chooses nested loop joins due to row estimation mismatches, often resulting in dramatically slower performance than hash or merge joins.
+
+**[Learn more about Inefficient Nested Loops →](/docs/query-advisor/insights/inefficient-nested-loops)**
+
+### Wrong Index Due To ORDER BY
+
+Identifies queries where Postgres uses an index primarily for sorting while filtering large numbers of rows, indicating that a different index strategy would be more efficient.
+
+**[Learn more about Wrong Index Due To ORDER BY →](/docs/query-advisor/insights/wrong-index-order-by)**
+
+## Optimization techniques
+
+Query Advisor employs several proven Postgres optimization techniques:
+
+### Query rewrites
+- **Materialized CTEs**: Forces Postgres to materialize intermediate results to prevent inefficient nested loops
+- **Sort prevention**: Uses arithmetic expressions (e.g., `+0`) to prevent Postgres from using index sort order
+
+## Detection methodology
+
+Query Advisor analyzes EXPLAIN plans through a **two-phase process**:
+
+### Phase 1: Initial screening
+- Processes **7 million+ plan samples daily** in real-time
```

Comment:
I'm not sure if this number is needed. Something like this can be outdated easily, also the customer may care about how their query is analyzed, but maybe don't care much about how the pganalyze app analyzes other customers' query, especially in this context? I think this is here to lead to the "scalable" part in the line 57... maybe something like this might fit better in the blog post than the documentation?